disnake/enums.py

```diff
@@ -505,6 +505,7 @@ class UserFlags(Enum):
     verified_bot_developer = 131072
     discord_certified_moderator = 262144
     http_interactions_bot = 524288
+    active_application = 16777216
```

Comment:
this is actually part of ApplicationFlags

Reply:
fixed